src/widgets/Survey/ui/PassingScreen.tsx

```diff
@@ -277,6 +288,39 @@ const PassingScreen = () => {
     onTimerEnd: hasNextStep ? onNext : openSubmitModal,
   });
 
+  const IdleTimer = useIdleTimer({
+    time: context.event.timers.idleTimer, // Value should be provided from event DTO, if null - timer will not work
```

Comment:
The comment is nice for understanding the use case. However, the same thing can be accomplished by writing the code. You can either move `time` to the `start` function and explicitly check if `time` is not null (as it was mentioned earlier) OR explicitly has a flag, for instance "enabled":
```
const IdleTimer = useIdleTimer({
   time: context.event.timers.idleTimer,
   enabled: !!context.event.timers.idleTimer
   onFinish: () => {
      if (props.onTimerFinish) {
        props.onTimerFinish();
      }
    },
});
```

Reply:
Yes, good suggestion. Done